examples/todoapp/lib/lib_permissions/data_sources/local/permissions_local_data_source.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:convert';
+
+import '../../../app_extensions.dart';
+import '../../../base/data_sources/local/shared_preferences_instance.dart';
+
+class PermissionsLocalDataSource {
+  final SharedPreferencesInstance sharedPreferences;
+
+  PermissionsLocalDataSource({required this.sharedPreferences});
+
+  Future<bool> savePermissions(Map<String, bool> permissions) async {
+    return await sharedPreferences.setString(
```

Comment:
Considering we use realm, why the permissions are stored in the shared preferences?

Reply:
Since we already have shared preferences in place, it seemed easier to store it this way, instead of creating a realm model for each of the classes that will be stored

Reply:
The intent of preferences is to.... store user preferences such as preferred language etc. I think when it comes to permissions we can't use the shared preferences. The other advantage by using Mongo Atlas for permissions is that in the future we can implement real-time synchroisation and the user should not restart its app in order to see the changed permissions.

Reply:
Done